plugin.js

```diff
@@ -90,17 +108,33 @@ const buildRouter = (admin, predefinedRouter) => {
  * Using the router requires you to install `express-session` as a
  * dependency.
  *
- * @param  {AdminBro} admin                    instance of AdminBro
- * @param  {Object} auth                          authentication options
- * @param  {Function} auth.authenticate           function takes 2 arguments: email
- *                                                and password. Returns authenticated
- *                                                user or null, in case of a wrong email
- *                                                and/or password
- * @param  {String} auth.cookiePassword           secret used to encrypt cookies
- * @param  {String} auth.cookieName=adminbro      cookie name
- * @param  {express.Router} [predefinedRouter]    Express.js router
- * @param  {session.options} [sessionOptions]     Options that are passed to express-session
- * @return {express.Router}                       Express.js router
+ * @param  {AdminBro} admin
```

Comment:
please leave the formatting as it was before.

Reply:
But it exceeds line length and throws eslint errors.